python/tk_desktop/banner_widget.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+from sgtk.platform.qt import QtCore, QtGui
+from .ui.banner_widget import Ui_BannerWidget
+
+
+logger = sgtk.platform.get_logger(__name__)
+
+
+class BannerWidget(QtGui.QWidget):
+    """
+    Shows a notification in the banner.
+
+    :signals: dismissed() Invoked when the banner is dismissed by the user.
+    """
+
+    dismissed = QtCore.Signal(object)
+
+    def __init__(self, mgr, notif, parent=None):
+        """
+        :param mgr: ``NotificationsManager`` instance.
+        :param notif: ``Notification`` instance to display.
```

Comment:
personally prefer `notification` here as opposed to the abbreviation. I think it will make the rest of the code clearer.